src/PdfSharper/Pdf.AcroForms/PdfAcroForm.cs

```diff
@@ -44,7 +44,7 @@ internal PdfAcroForm(PdfDocument document)
             _document = document;
         }
 
-        internal PdfAcroForm(PdfDictionary dictionary)
+        public PdfAcroForm(PdfDictionary dictionary)
```

Comment:
Let's remove this public constructor, I believe all pdfs may have a form it just may be empty. I don't want to give the impression that we support adding  a form to a document.